tests/components/text/test_device_action.py

```diff
@@ -100,7 +100,7 @@ async def test_get_actions_hidden_auxiliary(
             "entity_id": entity_entry.id,
             "metadata": {"secondary": True},
         }
-        for action in ["set_value"]
+        for action in ("set_value",)
```

Comment:
single item loop

Reply:
I think for this one it is a "standard" loop in all the platform tests (see #119463) and I would prefer not to diverge

Reply:
ok, sounds reasonable 👍